src/BlazorWebView/src/SharedSource/WebView2WebViewManager.cs

```diff
@@ -154,6 +156,9 @@ BlazorWebViewHandler blazorWebViewHandler
 		)
 			: base(services, dispatcher, AppOriginUri, fileProvider, jsComponents, hostPagePathWithinFileProvider)
 		{
+
```

Comment:
```suggestion
```

Reply:
Normally I would not care about extra lines, but the tests failed so sure

(That's a joke, I would have merged it anyway)